spec/jobs/notify_email_job_spec.rb

```diff
@@ -0,0 +1,172 @@
+# frozen_string_literal: true
+
+RSpec.describe NotifyEmailJob, type: :job do
+  subject(:perform!) { described_class.new.perform(notification_id: notification.id) }
+
+  let(:perform_ignore_errors!) { perform! rescue nil }
```

Comment:
it looks like `ignore_errors` is the action here,
maybe
`perform_ignoring_errors!`
or 
`perform_and_ignore_errors!`
 are better name.

Reply:
ok, have renamed to `perform_and_ignore_errors!`